### PR TITLE
DATAUP-266 staging refresh

### PR DESCRIPTION
**Pull request by briehl**
# Description of PR purpose/changes

As reported in DATAUP-266, when a user finishes uploading a file, the file viewer doesn't refresh automatically anymore. This restores that functionality. I also noticed some browser slowdown / lockup when uploading a lot of small files at one time, which trigger a refresh (and a `list` call to the staging service) when each file ends. If the files are small enough, or the user's upload speed is fast enough, this might cause problems. So the refresh is only allowed to run up to once per second.

A side effect of this is setting the local `config.json` to use the `dev` environment. This is currently needed to be able to run tests against CI, and will have no effect on either the built image or deployment.

# Jira Ticket / Issue #
e.g. https://kbase-jira.atlassian.net/browse/DATAUP-X
- [x] Added the Jira Ticket to the title of the PR e.g. (DATAUP-69 Adds a PR template)

# Testing Instructions
* Details for how to test the PR: 
Just upload a file, and it should refresh and appear right away.
- [x] Tests pass in travis and locally 
- [x] Changes available by spinning up a local narrative and navigating to _X_ to see _Y_

# Dev Checklist:

- [x] My code follows the guidelines at https://sites.google.com/truss.works/kbasetruss/development
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules
- [x] I have run Black and Flake8 on changed Python Code manually or with git precommit (and the travis build passes)

### CODE REVIEW

File: kbase-extension/static/kbase/js/widgets/narrative_core/kbaseNarrativeStagingDataTab.js [hunk 1]
```diff
@@ -65,38 +68,73 @@ define([
...
         render: function() {
             return this.getUserInfo()
-            .then(userInfo => {
-                var $mainElem = $('<div>')
-                    .css({
-                        'height': '604px',
-                        'padding': '5px',
-                        'overflow-y': 'auto'
+                .then(userInfo => {
+                    const $mainElem = $('<div>')
```
Comment (ialarmedalien) [inline kbase-extension/static/kbase/js/widgets/narrative_core/kbaseNarrativeStagingDataTab.js:72]: could you give this a class and put the style info into a css file?

File: test/unit/spec/narrative_core/kbaseNarrativeStagingDataTab-spec.js [hunk 2]
```diff
@@ -60,30 +56,109 @@ define([
...
+            const userInfo = await stagingWidget.getUserInfo();
+            expect(userInfo).toEqual({user: fakeUser, globusLinked: false});
+        });
+
+        it('gets user info and parsed into whether or not the user is linked to globus', async () => {
+            const $dummyNode = $('<div>'),
+                stagingWidget = new StagingDataTab($dummyNode);
+            const userInfo = await stagingWidget.getUserInfo();
+            expect(userInfo).toEqual({user: fakeUser, globusLinked: true});
+        });
```
Comment (ialarmedalien) [inline test/unit/spec/narrative_core/kbaseNarrativeStagingDataTab-spec.js:91]: extra tests! 🎉

**Review by ialarmedalien — APPROVED** [review]
LGTM

File: kbase-extension/static/kbase/js/widgets/narrative_core/kbaseNarrativeStagingDataTab.js (hunk 1 above)
Comment (briehl) [reply kbase-extension/static/kbase/js/widgets/narrative_core/kbaseNarrativeStagingDataTab.js:72]: Good idea. I mostly went through and fiddled with layout and minor linting in that file, but didn't adjust CSS. I'll put that in a class.

Comment (briehl) [reply kbase-extension/static/kbase/js/widgets/narrative_core/kbaseNarrativeStagingDataTab.js:72]: Ok, done.

File: kbase-extension/static/kbase/css/kbaseNarrative.css [hunk 3]
```diff
@@ -88,6 +88,12 @@
   cursor: pointer;
 }
 
+.kb-data-staging__container {
```
Comment (ialarmedalien) [inline kbase-extension/static/kbase/css/kbaseNarrative.css:91]: 👍

**Pull request merged**